### DISCUSSION TITLE
Help troubleshooting - iPhone USB connection with iTunes

### DISCUSSION THREAD

**Discussion by rorycaputo** [sandboxie-plus/Sandboxie#2944] [Q&A Feedback]
Hello,

I have a need to connect an iPhone via USB to iTunes to transfer some software but I'm trying to avoid the bloatware of iTunes & iCloud installed on my Windows machine.

I was able to get iTunes successfully installed & running in Sandboxie Plus, and I believe I am able to access my USB-connected iPhone as well. However I haven't been able to get iTunes to notice the device.

I'm wondering what the first steps might be to debug/troubleshoot the problem and what my chances of solving this issue might be. Apple does a lot of proprietary 'connection handling' via AppleMobileDeviceHelper etc so I wonder if any of that is interfering.

Is there a good way to make sure my device is working properly in Sandboxie outside of iTunes? Is this a topic that other users have explored in the past?

Thanks in advance.

**Comment by isaak654** [comment]
> I was able to get iTunes successfully installed & running in Sandboxie Plus

Sorry to bother, may I ask how to do this exactly? With the steps provided in #1767, I couldn't do it.

**Reply by rorycaputo** [reply]
I used the DefaultBox but with these options added:
```
ExposeBoxedSystem=y
MsiInstallerExemptions=y
```

Also I made sure to download the iTunes installer straight from Apple, not the Microsoft Store version:
https://support.apple.com/downloads/itunes

(And I think I still had to run the installer multiple times to get a successful install) Hope this helps.

<img width="682" alt="Screenshot 2023-05-23 084748" src="https://github.com/sandboxie-plus/Sandboxie/assets/22970205/3644469c-ef4e-4589-86bb-692cda0fd080">

**Reply by Ouro17** [reply]
Hello, I'm trying to install iTunes on the sandboxie but I get a error message related to the AppleMobileDevice.exe service

```
AppleMobileDevice.exe SBIE2211 Sanboxed service failed to start: apple mobile device service
```
After that, the iTunes installer just revoque itself and finish the installation with errors.

Did you have this error?

**Reply by e-t-l** [reply]
@rorycaputo did you ever figure out how to get it to recognize your device?
